Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
```

Comment:
In short, PHPunit looks for functions starting with 'test' and then for additional annotations such as 'depends'.

https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html